curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e74d0fb4f0687bf4145d.md

```diff
@@ -22,7 +22,7 @@ assert.match(code, /const\s+taskData\s*=\s*\[\];?/)
 You should use `let` to create an empty `currentTask` object.
 
 ```js
-assert.match(code, /let\s+currentTask\s*=\s*\{\};?/)
+assert.match(code, /let\s+currentTask\s*=\s*\{\s*\};?/)
```

Comment:
This works, but we really should refactor a lot of these tests to rely less on strict regex tests and better utlize the other assert methods that come with chai

Read through my message, that I left here to make the following changes so the tests are more granular and rely lesson pure regex solutions 

https://github.com/freeCodeCamp/freeCodeCamp/issues/53351#issuecomment-1909194609

Reply:
By better utilizing the other assert methods, we won't run into these types of strict testing issues 👍

Reply:
whoops, totally missed that comment! I'll work on that now